src/com/sahokia/kos/concerttickets/service/TicketService.java

```diff
@@ -18,48 +25,66 @@ public class TicketService {
         POSSIBLE_HALLS.add("Extra Hall");
     }
 
-    public static void main(String[] args) {
-        for (int i = 0; i < 10; i++) {
-            TICKET_STORAGE.add(new Ticket(
-                    getRandomConcertHall(POSSIBLE_HALLS),
-                    getRandomEventCode(),
-                    RANDOM_GENERATOR.nextBoolean(),
-                    StadiumSector.values()[RANDOM_GENERATOR.nextInt(StadiumSector.values().length)],
-                    getRandomBackpackWeight(MIN_WEIGHT, MAX_WEIGHT),
-                    getRandomBigDecimal(MIN_PRICE, MAX_PRICE)
-            ));
-        }
-        for (Ticket ticket : TICKET_STORAGE) {
-            System.out.println(ticket);
+    private final List<Ticket> ticketStorage;
+
+    public TicketService() {
+        this.ticketStorage = new ArrayList<>();
+        this.setId(UUID.randomUUID().toString());
+    }
+
+    @Override
+    public void print() {
+        for (Ticket ticket : this.ticketStorage) {
+            ticket.print();
+            System.out.println("===========================================");
         }
-        handleTicket("hello");
-        handleTicket(TICKET_STORAGE.get(2).getId());
     }
 
-    public static void handleTicket(String id) {
+    public List<Ticket> getTicketStorage() {
+        return this.ticketStorage;
+    }
+
+    public void addTicket(Ticket ticket) {
+        this.ticketStorage.add(ticket);
+    }
+
+    public Ticket getRandomTicket() {
```

Comment:
from the method name it seems that we need to return one random ticket from our storage

Reply:
Renamed the method as "generateRandomTicket()"